.eslintrc.js

```diff
@@ -0,0 +1,143 @@
+module.exports = {
+    env: {
+        es6: true,
+        browser: true,
+    },
+    ignorePatterns: [
+        "**/dist/",
+        "**/htmlcov/**",
+        "**/node_modules/",
+        "*.config.js",
+        "*.json",
+        "*.lock",
+        ".eslintrc.js",
+        ".venv/",
```

Comment:
```suggestion
        ".venv/",
        "build/",
        "docs/",
```
Sometimes I build the package locally with `python setup.py sdist bdist_wheel`, so ignoring `build/` would help with that. And the documentation build in `docs/` also produced a lot of errors for me.

Reply:
yup, good point. Probably also exclude them from prettier?

So I'll just undo the escape-attempts and dismiss the complaints for now as wontfix?

Reply:
Yes, sounds good :+1: